lib/ansible/modules/cloud/vmware/vsphere_copy.py

```diff
@@ -15,49 +15,43 @@
 DOCUMENTATION = '''
 ---
 module: vsphere_copy
-short_description: Copy a file to a vCenter datastore
+short_description: Copy a file to a VMware datastore
 description:
-    - Upload files to a vCenter datastore
+    - Upload files to a VMware datastore through a VCenter of a ESXi
 version_added: 2.0
 author:
 - Dag Wieers (@dagwieers)
 options:
+  hostname:
+    version_added: "2.9"
+  port:
+    version_added: "2.9"
+  username:
+    version_added: "2.9"
   host:
     description:
-      - The vCenter server on which the datastore is available.
-    required: true
-    aliases: ['hostname']
```

Comment:
Not sure why we are removing alias and adding new parameter here.

Reply:
Because I thought they were dup because of the `vmware.documentation` fragment.